src/Microsoft.DotNet.Build.Tasks.Installers/src/GenerateMacOSDistributionFile.cs

```diff
@@ -44,7 +44,7 @@ public override bool Execute()
 function IsX64Machine() {
 var machine = system.sysctl(""hw.machine"");
 system.log(""Machine type: "" + machine);
-var result = machine == ""x64"" || machine.endsWith(""_x64"");
+var result = machine == ""x64"" || machine.endsWith(""_64"");
```

Comment:
Can we make this match https://github.com/dotnet/runtime/pull/59210/files#diff-c6e8d33ed66dda3512254f78a84b3551527f3f67c4c128263ac130cc4d34f3e6R99
```suggestion
var result = machine == ""amd64"" || machine== ""x86_64"";
```

Reply:
The original value here was based on me misreading a document that mentioned
`x86_64` (`x64`) .  I think that document was trying to communicate that it meant `x64` but was written as `x86_64` and I transposed the `x`.